app/models/group.rb

```diff
@@ -121,7 +121,7 @@ def inactive?
   end
 
   def point_eligible_memberships
-    memberships.includes(:user).select(&:active?)
+    memberships.includes(:user,:posts,:post_types).select(&:active?)
```

Comment:
Let's put some spaces next to the commas, for optimal reading experience.

Reply:
Done!